backend/src/app.js

```diff
@@ -58,11 +58,23 @@ router.get('/v1/templates/:name', (ctx) => {
         ctx.body = e;
         return;
     }
+
+    const readmeTemplatePath = path.join(ctx.templatePath, ctx.params.name, 'README.md');
     ctx.body = {
+        readme: getReadme(readmeTemplatePath),
         json_schema: jsonSchema
     };
 });
 
+function getReadme(templatePath) {
```

Comment:
@Thebigbignooby Have you try without readme? I think this will crash, no?

Reply:
@Its-Alex fixed.